src/css/component.nav-primary.css

```diff
@@ -2,6 +2,10 @@
 
 .nav-primary {
   --nav-primary-gutters-x: var(--size-3);
+  font-size: var(--font-size-md);
+  font-weight: var(--font-weight-normal);
+  letter-spacing: var(--font-tracking-normal);
```

Comment:
Admittedly some of this will be inherited from the baseline but it makes me feel more comfortable with explicit control, that's why I've added font declarations to most components.

Reply:
After some slack chat we've resolved this.

I've removed the `weight` and `letter-spacing` declarations, unless of course we need to override the baseline.